core/lib/spree/core/controller_helpers/gem_checking.rb

```diff
@@ -0,0 +1,26 @@
+module Spree
+  module Core
+    module ControllerHelpers
+      module GemChecking
```

Comment:
Maybe a name like `ComponentAvailability` (I don't like this one 100% but maybe there's something similar) would be more clear here?